samples/ImplicitFlow/AureliaApp/custom_typings/workarounds.d.ts

```diff
@@ -0,0 +1,3 @@
+interface ReadableStream {}
```

Comment:
Otherwise the TypeScript compiler will complain. Lemme know if you'd like more details.